assets/css/main.css

```diff
@@ -568,17 +568,63 @@ td.submissions-dropdown {
 .form-inline .form-group div {
     display: inline-block;
 }
-.correct-option {
-  margin-left: -38px;
-  margin-right: 20px;
-  color: green;
+
+/* Mooc-grader questionnaire */
+
+.exercise .aplus-quiz1 .question-area {
+    margin-top: 20px;
+    margin-bottom: 20px;
 }
-.neutral-option {
-  margin-left: -40px;
-  margin-right: 18px;
-  font-size: 18px;
-  color: #f5bc09;
+
+.exercise .aplus-quiz1 .question-description {
+    margin-bottom: 5px;
+}
+
+.exercise .aplus-quiz1 .error-text {
+    margin-top: 5px;
+    margin-bottom: 5px;
+    color: #a94442;
+    font-weight: bold;
+    margin-bottom: 0px;
+}
+
+.exercise .aplus-quiz1 .has-error .radio,
+.exercise .aplus-quiz1 .has-error .checkbox {
+    color: #333;
+}
+
+.exercise .aplus-quiz1 .option-checked {
+    font-weight: bold;
 }
+
+.exercise .aplus-quiz1 .hint {
+    display: block;
+    margin-top: 5px;
+    margin-bottom: 5px;
+    color: #737373;
+    font-style: italic;
+}
+
+.exercise .aplus-quiz1 .hint-checkbox {
+    padding-left: 20px;
+}
+
+.exercise .aplus-quiz1 .icon-correct {
```

Comment:
I would prefer these classes to contain all the details, so that the exercise service could just write `<i class="icon-correct" aria-hidden="true"></i>`. Hence, something like:

```css
.icon-correct {
    color: #5cb85c;
    /* glyphicon */
    position: relative;
    top: 1px;
    display: inline-block;
    font-family: 'Glyphicons Halflings';
    font-style: normal;
    font-weight: 400;
    line-height: 1;
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
}
.icon-correct:before {
    content: "\e013";
}

Reply:
Note that the glyphicon part would be easy to include with `@import` (or `@use`, I'm not sure right now) in SASS.

Reply:
Hmm.. Also, should these be `question-icon-ok` or `quiz1-icon-ok` :thinking:

Reply:
The generic stuff like form-group, form-control makes sense to support, glyphicons are something that might be interesting to replace with fontawesome at some point.

With the prefixes, I was thinking that they might easily collide with some other classes somewhere else. It would be wery possible to add classes like `icon-*` for some totally different purpose, and then those would effect these too. Basically, it would be beneficial if _all_ the classes would have `aplus-quiz1-` prefix, but that becomes very annoying.

Reply:
This doesn't seem to be well understoond problem in the internet. Bootstrap ignored to problem and materialui has an open issue about it. Some small and unknown stackoverflow about this https://softwareengineering.stackexchange.com/a/351554 .

Basically, every library whih adds css classes, should namespace those. In our case, that means the protocol support should namespace all classes. However, I assume that not so common words also work for the purpose well enough.

Reply:
This is interesting, and now I definitely understand the issue better. I changed those, and added the quiz1-prefix for the icons.